docs/preact-testing-library/api.md

```diff
@@ -0,0 +1,273 @@
+---
+id: api
+title: API
+---
+
+`preact-testing-library` re-exports everything from `dom-testing-library` as
+well as these methods:
+
+- [`render`](#act)
+- [`cleanup`](#cleanup)
+- [`act`](#act)
+
+---
+
+## `render`
+
+```typescript
+function render(
+  ui: React.ReactElement<any>,
+  options?: {
+    /* You won't often use this, expand below for docs on options */
+  }
+): RenderResult
+```
+
+Render into a container which is appended to `document.body`.
+
+```jsx
+import { render } from 'preact-testing-library'
+
+render(<div />)
+```
+
+```jsx
+import { render, cleanup } from 'preact-testing-library'
+import 'jest-dom/extend-expect'
+afterEach(cleanup)
+
+test('renders a message', () => {
+  const { container, getByText } = render(<Greeting />)
+  expect(getByText('Hello, world!')).toBeInTheDocument()
+  expect(container.firstChild).toMatchInlineSnapshot(`
+    <h1>Hello, World!</h1>
+  `)
+})
+```
+
+> Note
+>
+> The [cleanup](#cleanup) function should be called between tests to remove the
+> created DOM nodes and keep the tests isolated.
+
+## `render` Options
+
+You wont often need to specify options, but if you ever do, here are the
+available options which you could provide as a second argument to `render`.
+
+### `container`
+
+By default, `preact-testing-library` will create a `div` and append that div to
+the `document.body` and this is where your react component will be rendered. If
+you provide your own HTMLElement `container` via this option, it will not be
+appended to the `document.body` automatically.
+
+For Example: If you are unit testing a `tablebody` element, it cannot be a child
+of a `div`. In this case, you can specify a `table` as the render `container`.
+
+```jsx
+const table = document.createElement('table')
+
+const { container } = render(<TableBody {...props} />, {
+  container: document.body.appendChild(table),
+})
+```
+
+### `baseElement`
+
+If the `container` is specified, then this defaults to that, otherwise this
+defaults to `document.documentElement`. This is used as the base element for the
+queries as well as what is printed when you use `debug()`.
+
+### `hydrate`
```

Comment:
Don't see this option in https://github.com/antsmartian/preact-testing-library/blob/master/src/index.js